modules/go-health-monitoring.adoc

```diff
@@ -20,4 +20,6 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 . If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are:
```

Comment:
| Current | Suggested |
|-|-|
| If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are: | If the application is out of sync or degraded, the respective icons are displayed under the *Resources* <tab/section???>. To view the sync and health status of the applications, over the cursor over the icons. |
| For `out of sync`, the yellow yield icon is displayed. | For `out of sync` status, the yellow yield icon is displayed. |
| For `degraded`, the broken heart icon is displayed. | For `degraded` status, the broken heart icon is displayed. | 
| Click on the *Deployment History* tab to view the application deployment history. The page includes details such as, *Last deployment*, *Description* (commit message), *Environment*, *Outcome* (Succeeded/Failed), *Author* and *Revision*. | To view the deployment history of an application, click on the *Deployment History* tab. The page includes details such as: *Last deployment*, *Description* (commit message), *Environment*, *Outcome* (Succeeded/Failed), *Author*, and *Revision*. |

NOTE:
- Text formatting under the Suggested column has been done according to Asciidoc format; it may generate different rendered view for Markdown. If you are copying, copy the raw text.
- For the 2nd and 3rd rows, you might consider using inline images. See https://docs.asciidoctor.org/asciidoc/latest/macros/images/#inline-image-macro. However, I'm not sure about the best practice for doc maintenance and rendering; get a second pinion from a more experienced reviewer.
- For the last row, if the texts within parenthesis () are part of UI, make them bold. If they are not, I suggest not using them at all.